gremlin-go/driver/connection_test.go

```diff
@@ -656,6 +656,32 @@ func TestConnection(t *testing.T) {
 		resetGraph(t, g)
 	})
 
+	t.Run("Test Traversal. GetResultSet", func(t *testing.T) {
+		skipTestsIfNotEnabled(t, integrationTestSuiteName, testNoAuthEnable)
+
+		// Initialize graph
+		g := initializeGraph(t, testNoAuthUrl, testNoAuthAuthInfo, testNoAuthTlsConfig)
+		defer g.remoteConnection.Close()
+
+		resultSet, err := g.V().HasLabel(personLabel).Properties(nameKey).GetResultSet()
+		assert.Nil(t, err)
+		assert.NotNil(t, resultSet)
+		allResults, err := resultSet.All()
+		if err != nil {
+			return
+		}
+		var names []string
+		for _, res := range allResults {
+			assert.NotNil(t, res)
+			vp, err := res.GetVertexProperty()
+			assert.Nil(t, err)
+			names = append(names, vp.Value.(string))
+		}
+		assert.True(t, sortAndCompareTwoStringSlices(names, testNames))
+
+		resetGraph(t, g)
```

Comment:
I think this should be on line 664, above the `Close()`

Reply:
For resetGraph? Don't we only want to reset the graph at the end? The initialize graph already checks that the graph is empty then populates it with data.